notebooks/05_operationalize/als_movie_o16n.ipynb

```diff
@@ -92,8 +92,11 @@
     "from azureml.core.model import Model\n",
```

Comment:
I think this line is not needed anymore `from azure.mgmt.compute import ComputeManagementClient` could you please confirm?